aten/src/ATen/native/cuda/ForeachBinaryOpList.cu

```diff
@@ -51,8 +51,6 @@ void foreach_tensor_list_op_(TensorList tensors1, TensorList tensors2, Scalar al
 
 #define FOREACH_BINARY_OP_LIST(NAME, OP)                                                                    \
 void foreach_tensor_##NAME##_list_kernel_cuda_(TensorList tensors1, TensorList tensors2) {                  \
-    check_foreach_api_restrictions(tensors1, tensors2);                                                     \
```

Comment:
is check_foreach_api_restrictions called anywhere now?

Reply:
Yeah, in slow path. [ForeachOpsKernels.cpp](https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/ForeachOpsKernels.cpp#L8)